### PR TITLE
feat: Collect modules recursivly in `vi/config`

### PR DESCRIPTION
**Pull request by sveneberth**
The viur-shop has it's modules as submodules under a main `shop`. As you can see [here](https://github.com/viur-framework/viur-shop/blob/191b6a56e764a709170bcd1e3f1e27f1e3fa4c1b/src/viur/shop/shop.py#L45-L53).

So for example we have this path `/shop/address/list`.
It's perfectly routeable, but the vi doesn't know it because submodules are not collected. This PR adds a recursive walk through all the modules and collects them in `/vi/config`.

### CODE REVIEW

File: src/viur/core/render/vi/__init__.py
Comment (ArneGudermann) [inline src/viur/core/render/vi/__init__.py:10]: Import both as `user` ?

Comment (sveneberth) [inline src/viur/core/render/vi/__init__.py:10]:
```suggestion
from viur.core.render.vi.user import UserRender as user  # this import must exist!
```

**Review by phorward — DISMISSED** [review]
For me, this is okay, what's with the others? @ArneGudermann @XeoN-GHMB

**Review by ArneGudermann — DISMISSED** [review]

**Comment by phorward** [comment]
I'm waiting for the approval by @akelch to merge this.

**Comment by sveneberth** [comment]
@ArneGudermann and I are planning adjustments here and in other methods for 100% compatibility of the recursion. I have therefore changed the status to Draft.

**Review by phorward — APPROVED** [review]

**Review by akelch — APPROVED** [review]

**Pull request merged**
